components/econet/econet.cpp

```diff
@@ -542,6 +546,12 @@ void Econet::register_listener(const std::string &datapoint_id, int8_t request_m
   if (request_mod >= 0 && request_mod < request_datapoint_ids_.size()) {
     request_datapoint_ids_[request_mod].insert(datapoint_id);
     request_mods_ = std::max(request_mods_, (uint8_t) (request_mod + 1));
+    if (request_mod == 1 || request_mod == 2 || request_mod == 3) {
```

Comment:
Can you add a configurable map from request_mod to update interval at https://github.com/esphome-econet/esphome-econet/blob/c3fda1ba48ecbd5a2a9c998bb2dd449e5dfa0319/components/econet/__init__.py#L74 ? If a request_mod isn't present, use the default update interval, if it is, use that.

Reply:
Made this configurable now. Do want to change the format of the config noted in separate comment.